spec/linter-codeclimate-spec.js

```diff
@@ -13,14 +14,15 @@ describe('The codeclimate provider for Linter', () => {
 
     waitsForPromise(() =>
       Promise.all([
-        atom.packages.activatePackage('language-ruby'),
         atom.packages.activatePackage('linter-codeclimate'),
       ]),
     );
   });
 
   it('works with a valid .codeclimate.yml file', () =>
-    waitsForPromise(() =>
+    waitsForPromise(
+      { timeout: 10000 },
```

Comment:
For me it was failing without this. Reported time was a bit over 5 sec so I assume it's 5 seconds by default. Maybe this changed recently in some of the packages. 

Would you consider a yarn.lock or something else akin to that to lock dependency versions?